java/build/bazel/tests/integration/BazelBaseTestCase.java

```diff
@@ -17,13 +17,8 @@
 import org.junit.Before;
 import org.junit.BeforeClass;
 
-import java.io.File;
 import java.io.IOException;
-import java.util.List;
 
-/**
```

Comment:
Why did you remove this? I think the comment still holds

Reply:
`A base class to do integration test that call Bazel` - I don't feel like it adds useful information and it doesn't really call Bazel anymore.

Reply:
I think you’re misreading it. It’s a base class to do an integration test which calls bazel. The base doesn’t call it. The test calls it.

Reply:
Do you see value in this comment? Is it just project's style?